torch/_inductor/codegen/wrapper.py

```diff
@@ -1544,21 +1544,30 @@ def codegen_conditional(self, conditional):
 
     def codegen_while_loop(self, while_loop):
         name = while_loop.get_name()
-        outer_inputs = [buf.codegen_reference() for buf in while_loop.operands]
+        outer_carried_inputs = [
+            buf.codegen_reference() for buf in while_loop.carried_inputs
+        ]
+        outer_additional_inputs = [
+            buf.codegen_reference() for buf in while_loop.additional_inputs
+        ]
+        outer_inputs = outer_carried_inputs + outer_additional_inputs
```

Comment:
Nit: could as well combine `while_loop.carried_inputs + while_loop.additional_inputs` in the pre-existing list comprehension, as the same is done in both cases.

Reply:
Sure. Thanks for pointing out.